packages/terra-form/docs/Input.md

```diff
@@ -0,0 +1,16 @@
+# Input
+
+Generic input which represents an HTML input element directly.
+
+## Usage
+
+```jsx
+import React from 'react';
+import { Input } from 'terra-form';
+
+<Input
+  required={false}
```

Comment:
Our linter prefers us to use the top approach you have when the boolean value is known to be true. Looks like I forgot to update the documentation to reflect the change. I'll get the MD fixes.

As for the is prefix, I feel like the solution is situational. required is a standard html attribute, so for a variable like required, I think keeping it as it would be the ideal approach. For something like isInline, inline isn't a mapped html attribute, so I feel that the prefix would be needed there.